### PR TITLE
Add numfocus role

### PR DESCRIPTION
**Pull request by ivirshup**
Adding a NumFOCUS role to our governance docs.

This is a draft for now, to let us discuss naming and how we specify the role.

### CODE REVIEW

File: roles.md [hunk 1]
```diff
@@ -105,6 +105,15 @@ The scientific advisory boards consists of highly experienced scientists devoted
 
 Members of the scientific adivsory boards are nominated by the core team. 
 
+## NumFOCUS subcommittee
+
+The core team will maintain a subcommittee to manage its interactions with NumFOCUS.
+
+* The NumFOCUS Subcommittee is comprised of 5 persons who manage project funding that comes through NumFOCUS. It is expected that these funds will be spent in a manner that is consistent with the non-profit mission of NumFOCUS and the direction of the Project as determined by the full Council.
```
Comment (Zethson) [inline roles.md:112]: This does NOT need to be restricted to 5 people. IIRC it was 5+. I know that a limit of 5 people can speed up some processes but I think that it's pretty difficult to always justify who shall be in the subcommittee and who is not (even though we don't grant these people additional power).

` the full Council`

This would mean only Isaac, Danila + me? I thought that we had the final say but not the ruling power. Maybe rephrase this to core team?

File: roles.md [hunk 2]
```diff
@@ -105,6 +105,15 @@ The scientific advisory boards consists of highly experienced scientists devoted
...
+* This Subcommittee shall NOT make decisions about the direction, scope or technical direction of the Project.
```
Comment (Zethson) [inline roles.md:113]: Could add that the core team or the current governance structure as defined somewhere else makes these decisions.

But generally this is good!

**Review by Zethson — CHANGES_REQUESTED** [review]

File: roles.md (hunk 1 above)
Comment (grst) [reply roles.md:112]: agree this should read "core team"

Comment (ivirshup) [reply roles.md:112]:
```suggestion
* The NumFOCUS Subcommittee is comprised of at least 5 persons who manage project funding that comes through NumFOCUS. It is expected that these funds will be spent in a manner that is consistent with the non-profit mission of NumFOCUS and the direction of the Project as determined by the Core Team.
```

**Review by Zethson — APPROVED** [review]

**Review by grst — APPROVED** [review]
